scripts.json

```diff
@@ -1,5 +1,6 @@
 {
     "vendor": [
+        "https://cdn.socket.io/socket.io-1.3.4.js",
         "app/components/angular/angular.js",
```

Comment:
I think this will fail in the build, you should instead try to mock this api if your tests are dependant on it